### ISSUE TITLE
代码

### ISSUE THREAD

**Issue by staring-s** [CoinCheung/BiSeNet#159]
大佬，关于demo里面的代码，这一步的处理pred = palette[out]没弄明白。
out = net(im)[0].argmax(dim=1).squeeze().detach().cpu().numpy()
此时的out为(1024,2048)
pred = palette[out]
pred为(1024,2048,3)
此时为pred通道方式为HWC。
单看这一步palette = np.random.randint(0, 256, (256, 3), dtype=np.uint8)
是返回随机整数，pred = palette[out]然后就不明白了。

**Comment by CoinCheung** [comment]
意思是给生成的pred上色，不同类别随机分配不同的颜色

**Comment by sky999syn666** [comment]
> 大佬，关于demo里面的代码，这一步的处理pred = palette[out]没弄明白。
> out = net(im)[0].argmax(dim=1).squeeze().detach().cpu().numpy()
> 此时的out为(1024,2048)
> pred = palette[out]
> pred为(1024,2048,3)
> 此时为pred通道方式为HWC。
> 单看这一步palette = np.random.randint(0, 256, (256, 3), dtype=np.uint8)
> 是返回随机整数，pred = palette[out]然后就不明白了。
请问demo.py你可以跑通吗，我跑的时候会报错，如下：
![image](https://user-images.githubusercontent.com/46163165/133734938-482b7e7d-6030-48ab-b7df-30df63a008db.png)
请问你知道这里哪里出问题了吗
